### DISCUSSION TITLE
Compile for MACOS?

### DISCUSSION THREAD

**Discussion by lkrynicky** [seerge/g-helper#2024] [Ideas]
Hi everyone,

     Do you think it would be possible to compile this for Mac OS? I am newer to CS, I haven't done much in compiling programs for different operating systems. I am guessing the UI is windows specific which wouldn't be possible to run on Mac OS? Would it be possible to port it over? 
     
     Thank you!

**Comment by seerge** [comment]
@lkrynicky i don't think it will be possible, as app depends on Asus Control Interface driver for all it's actions. It exists for windows. 
It also uses lots of windows API calls.

What is the purpose of compiling app for MacOS ? As far as I remember you can't run it properly on a most of Asus laptops anyway :)

**Reply by lkrynicky** [reply]
Thanks @seerge for getting back to me, just so ASUS ROG peripherals could but more functional with MacOS but thank you for clearing it up for me!

**Reply by seerge** [reply]
@lkrynicky it would be way easier to launch G-Helper inside Virtualbox or Parallels then. Just make sure that virtual machine has access to real USB

**Reply by lkrynicky** [reply]
@seerge Thank you for the advice, I will give it a try!
